receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -422,10 +423,9 @@ func TestScrapeMetrics_Filtered(t *testing.T) {
 
 			handles := make([]*processHandleMock, 0, len(test.names))
 			for i, name := range test.names {
-				handleMock := newDefaultHandleMock()
+				handleMock := newDefaultHandleMock(time.Now().UnixMilli() - test.upTimeMs[i])
 				handleMock.On("Name").Return(name, nil)
 				handleMock.On("Exe").Return(name, nil)
-				handleMock.On("CreateTime").Return(time.Now().UnixMilli()-test.upTimeMs[i], nil)
```

Comment:
Is this change really needed or we can keep? Does it change anything?

Reply:
It can work as it was previously. Undid the change